ecs-init/cache/cache.go

```diff
@@ -70,8 +70,9 @@ func (d *Downloader) fileNotEmpty(filename string) bool {
 // getRegion finds region from metadata and caches for the life of downloader
 func (d *Downloader) getRegion() string {
 	if d.region == "" {
-		region, ok := d.metadata.Region()
-		if ok != nil {
+		region, err := d.metadata.Region()
+		if err != nil {
+			log.Debug("Could not retrieve EC2 Metadata. %s", err)
```

Comment:
Can change the error message to be `"Could not retrieve the region from EC2 Instance Metadata"`? Also, this should be a warning, not debug.